### DISCUSSION TITLE
Using config parameter latest_stable_version as part of shortcode parameter?

### DISCUSSION THREAD

**Discussion by elevran** [google/docsy#1937] [Q&A] [answered]
I want to pass a URL value to a shortcode. The URL value depends on a parameter defined in the `config.toml`.
Specifically, I’m using Docsy theme with versioning and to have a target URL passed via a shortcode parameter to refer to `/docs/{{< param latest_stable_version >}}/overview` and evaluate when rendering to (e.g.,) `/docs/v1.0.1/overview` based on the parameter value in the config.
Using `{{< param latest_stable_version >}}` inside the short code parameter does not work and it renders the actual text as is.
Is it possible to pass an evaluated parameter (i.e., its value) to a shortcode?

**Comment by fekete-robert** [comment, marked as answer]
Oh, I see that you already got a response on the hugo forum, adding a link for reference: https://discourse.gohugo.io/t/using-parameters-in-shortcode-parameters/49337/4
